mappings/net/minecraft/world/gen/trunk/StraightTrunkPlacer.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_5140 net/minecraft/world/gen/trunk/StraightTrunkPlacer
+	METHOD <init> (III)V
+		ARG 1 baseHeight
+		ARG 2 firstRandomHeight
+		ARG 3 secondRandomHeight
```

Comment:
weren't these referred to as `heightRandA` and `heightRandB` before?

Reply:
heightRandA and heightRandB are not descriptive and doesn't fit with yarn guideline

Reply:
alright